packages/design-system/src/components/IdleTimeout/IdleTimeout.tsx

```diff
@@ -186,6 +186,11 @@ const IdleTimeout = ({
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
I tried adding the props as dependencies to the `useCallback` of `resetTimeouts`, but it doesn't work how useEffect works. The `resetTimeouts` function would only trigger on re-render and I'm not sure why... so this is my workaround

Only downside here is that this effect runs on mount so technically `setTimeoutCookies` gets called twice. Once in the above effect & once here

Reply:
I think that's fine for now. This is kind of a complex little component we've got here, and maybe over time we'll see ways to simplify it. For now, some comments could help a lot to explain the different useEffects and such